### PR TITLE
Make `meta_data_key` a kwarg

### PR DESCRIPTION
**Pull request by this-josh**
This allows for pandas formatting of other functions such as `WTI`

### CODE REVIEW

**Comment by this-josh** [comment]
These fails seem to be to do with sorting the index, which would seem preferable. Unless there's a reason to not sort?

File: alpyvantage/data_formating.py [hunk 1]
```diff
@@ -46,5 +46,11 @@ def _format_to_pandas(call_response, data_key):
     data_pandas.index.name = 'date'
     # convert to pandas._libs.tslibs.timestamps.Timestamp
     data_pandas.index = pd.to_datetime(data_pandas.index)
-    data_pandas.columns = [col.split('. ')[1] for col in data_pandas.columns]
+    data_pandas.sort_index(inplace=True)
```
Comment (this-josh) [inline alpyvantage/data_formating.py:49]: Here is the sort

Comment (gboehl) [reply alpyvantage/data_formating.py:49]: Thanks for the PR! I'm happy to merge. Yes, sorting seems preferable. I'll fix the tests...

**Comment by gboehl** [comment]
Good work, thanks a lot!

**Pull request merged**